README.md

```diff
@@ -311,7 +301,7 @@ Check the OpenAPI definitions for tasks data endpoints: [Monocle OpenAPI][monocl
 ```YAML
 workspaces:
   - name: default
-    crawlers_api_key: 1a2b3c4d5e
+    crawlers_api_key: API_KEY_ENV
```

Comment:
So I guess we can omit this crawlers_api_key as it is optional and then the backend is looking for CRAWLERS_API_KEY env var.